docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -270,8 +270,9 @@ The markers layer allows you to display data points as different marker shapes s
 
 ![Markers Layer](/static/img/docs/geomap-panel/geomap-markers-8-1-0.png)
 
-![Markers Layer Options](/static/img/docs/geomap-panel/geomap-markers-options-8-1-0.png)
+![Markers Layer Options](/media/docs/grafana/panels-visualizations/geomap-markers-options-11-1-0.png)
```

Comment:
The image is quite large when I look at it in preview so I've swapped out a regular markdown image line with a figure shortcode so we can control the size.
```suggestion
{{< figure src="/media/docs/grafana/panels-visualizations/geomap-markers-options-11-1-0.png" max-width="350px" alt="Markers layer options" >}}
```